app/views/admin/members/_legal.en.html.erb

```diff
@@ -1,6 +1,6 @@
 <p>
   This is a legal document. The secretary of
-  <%= MembershipRegister.society_name %>
+  <%= ENV.fetch("SOCIETY_NAME") { "Sample Society" } %>
```

Comment:
Why not change the implementation of `MembershipRegister.society_name`, or put this in `en.yml`? It's likely we might want to reuse this in multiple places.

Reply:
> Why not change the implementation of `MembershipRegister.society_name`, or put this in `en.yml`? It's likely we might want to reuse this in multiple places.

I prefer not to have that in a locale, because I believe the initial idea was to make it the most generic as possible, so other societies could use :).

Whad do you think?

Reply:
I don't mind, but here's a couple of alternatives:

```ruby
module MembershipRegister
  def self.society_name
    ENV.fetch("SOCIETY_NAME") { "Sample Society" }
  end
end
```

or…

```yaml
# en.yml
society_name: <%= ENV.fetch("SOCIETY_NAME") { "Sample Society" } %>
```

Consider the duplication in the following examples:

```erb
Hello and welcome to <%= ENV.fetch("SOCIETY_NAME") { "Sample Society" } %>.

The <%= ENV.fetch("SOCIETY_NAME") { "Sample Society" } %> has <%= member_numbers %> members.

Email from <%= ENV.fetch("SOCIETY_NAME") { "Sample Society" } %>: ....
```

It's a bit simpler and more readable to have:

```erb
Hello and welcome to <%= society_name %>.

The <%= society_name %> has <%= member_numbers %> members.

Email from <%= society_name %>: ....
```

Reply:
Done, but to tell the truth, I am not happy with it. It is used only once.

If it was used for more than one place I would already had done that.